ModelBundle/Tests/Manager/NodeManagerTest.php

```diff
@@ -59,4 +75,19 @@ public function provideLanguageAndSite()
             array('en', '2'),
         );
     }
+
+    /**
+     * test duplicateNode
+     */
+    public function testduplicateNode()
+    {
+        $nodeId = 'fakeNodeId';
+        $siteId = 'fakeSiteId';
+        $language = 'fakeLanguage';
+        $statusId = 'fakeStatusId';
+
+        $node = $this->manager->duplicateNode($nodeId, $siteId, $language, $statusId);
```

Comment:
there are multiple call that have been done in the method,

I think that you should make some verification !